### ISSUE TITLE
HTML formatting and max 1 attr per line

### ISSUE THREAD

**Issue by thedamon** [chrisvfritz/vue-enterprise-boilerplate#192]
The [linting docs](https://github.com/chrisvfritz/vue-enterprise-boilerplate/blob/master/docs/linting.md) say

> HTML (in templates and JSX) is linted by ESLint

As far as I can tell, it is being formatted by prettier as well.

I was trying to use this setup retroactively in my own project because of this specific line.. happy to see a fully working lint/editor format setup _without_ prettier because that would mean I can force attributes onto their own lines as [recommended in the official styleguide](https://vuejs.org/v2/style-guide/#Multi-attribute-elements-strongly-recommended).

Regrettably that doesn't seem to be happening for me (prettier is cramming attributes based on line length). Is that expected behaviour?

**Comment by chrisvfritz** [comment]
This is indeed expected behavior. I actually wrote that recommendation in the official styleguide, but since Prettier has no option to change this particular behavior, I personally find the consistency Prettier provides to be of greater benefit than this individual rule. You _could_ gain maximum control by running ESLint (and any other linters you want) again after Prettier to override anything you don't like, but I don't find that slowdown in linting to be worth it personally.

**Comment by thedamon** [comment]
Thanks. I'm unendingly frustrating that prettier isn't on board to add that option because I think it makes lots of sense to do so. Probably worth just running with them until another option comes along.

**Comment by escapedcat** [comment]
We just went through this process as well. We didn't use prettier and then we started.  
At first I was pretty annoyed by this as well. Prettier has its own rule when to put attributes on one line and when to split them into separate lines. Without checking the rule I would say it's between 3-4 attributes.  
By now we feel pretty comfortable with this. Overall we prefer the gain we get by using prettier instead of not using it. 

TL;DR We agree with Chris' opinion on this ;)

Just wanted to share our own experience with this.
